src/executor/LanceurExecutor.java

```diff
@@ -32,7 +32,7 @@ public Integer call() throws Exception {
 		try {
 			Integer retour = f.get();
 			System.out.printf("j'ai eu le retour %s%n", retour);
-
+                        System.out.println("Coucou sonar !");
 		} catch (InterruptedException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)